G1ANT.Addon.Net/G1ANT.Addon.Net/NetAddon.cs

```diff
@@ -27,5 +26,37 @@ namespace G1ANT.Addon.Net
     [CommandGroup(Name = "vnc",  Tooltip = "A command connected with a VNC server.")]
     public class NetAddon : Language.Addon
     {
+        public override void Initialize()
+        {
+            base.Initialize();
+        }
+
+        public override void LoadDlls()
+        {
+            UnpackDrivers();
+            base.LoadDlls();
+        }
+
+        private void UnpackDrivers()
+        {
+            var unpackfolder = AbstractSettingsContainer.Instance.UserDocsAddonFolder.FullName;
+            Dictionary<string, byte[]> exeList = new Dictionary<string, byte[]>()
```

Comment:
`exeList` - the name is misleading: is it a driver (vide `UnpackDriver`) or an exe (vide `exeList`)? Is it a dictionary or a list? => rename to `drivers`.
Don't use full type name when it does not bring any useful information => var.

Did you verify the performance/memory penalty for referencing the resource? If there's any then you should rewrite the code to reference only the resources that need to be saved to user disk (see my next comment).

Reply:
Its just copy pasted from other addon.
Than we need to change other addons also.

Reply:
> Its just copy pasted from other addon.

That's not a reason at all.

It would be great to update other addons, but it needs to be done in a separate task.

Reply:
https://github.com/G1ANT-Robot/G1ANT.Addon/blob/develop/G1ANT.Addon.Selenium/G1ANT.Addon.Selenium/SeleniumAddon.cs

Reply:
Made separate task for selenium addon: https://g1ant.manuscript.com/f/cases/7478/Addon-Selenium-normalize-resource-files-creation